autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -1,36 +1,110 @@
 import * as React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import {
-  List, Typography, Checkbox, Button, Menu, MenuItem, IconButton, Tooltip,
+  List, Typography, Checkbox, Button, Menu, MenuItem, IconButton,
+  Tooltip, ExpansionPanel as ExpansionPanelBase, ExpansionPanelSummary as ExpansionPanelSummaryBase,
+  ExpansionPanelDetails as ExpansionPanelDetailsBase,
```

Comment:
Use `Accordion` instead of `ExpansionPanel` for these components:
![image](https://user-images.githubusercontent.com/28655462/121787716-6b8a8700-cb8d-11eb-9fdd-c5c8350d65e2.png)

Reply:
Our version of material ui seems super outdated, Accordion doesn't exist yet lol.